blog/zh/blog/2022/07/29/why-we-need-apache-apisix.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "有了 NGINX 和 Kong，为什么还需要 Apache APISIX？"
+author: "韩飞"
+authorURL: "https://github.com/hf400159"
+authorImageURL: "https://github.com/hf400159.png"
```

Comment:
need to add credit for yuansheng as well

Reply:
This is a new article.